Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml

```diff
@@ -410,7 +410,8 @@ tasks:
         - operator: match
           left:
             value:
-              simple: File.Extension
+              simple: inputs.File.Extension
+            iscontext: true
           right:
             value:
               simple: .*(?:SH|JS|7Z|PY|DOC|RTF|XLS|PPT|PPS|PS1|BAT|EML|EXE|ISO|JAR|TXT|ODT|ODP|ODS|SWF|MSG|MSI|URL|PDF|RAR|VBS|ZIP|CAB|INK|XML|DLL|TAR|HTA|ELF|DOCX|DOCM|LINK|XLSX|XLSM|XLSB|PPTX|PPAM|HTML)\b
```

Comment:
Why not adding the "WORD" to here also? same thing will happened in this playbook.